In this notebook we will look at general 2-D motion problem under free fall conditions. First we will identify our initial position, velocity and angle of launch.

In [1]:
import numpy as np
import math
from sympy import *
init_printing(use_unicode = True)

In [2]:
x_i = 0.0
y_i = 40.0
v_i = 4.00 
theta = 35.0 
print('Initial velocity is ', v_i ,'meters per second')
print('Launch angle is ', theta ,'degress')

Initial velocity is  4.0 meters per second
Launch angle is  35.0 degress


Now we can use some facts about motion in two dimensions that are useful:

1. The acceleration in the x-direction is zero.
2. The acceleration in the y-direction is -9.81 m/s$^2$.

In [5]:
a_x = 0 
a_y = -9.81 
print('The acceleration in the x-direction is a_x = ' '%0.3f' %  a_x, 'meters per second squared')
print('The acceleration in the y-direction is a_y = ' '%0.3f' % a_y, 'meters per second squared')

The acceleration in the x-direction is a_x = 0.000 meters per second squared
The acceleration in the y-direction is a_y = -9.810 meters per second squared


Let us find the horizontal and vertical components of the velocity

In [6]:
v_ix = v_i*np.cos(theta*np.pi/180)
v_iy = v_i*np.sin(theta*np.pi/180)
print('The initial condition in the x-direction is v_ix = ' '%0.3f' % v_ix, 'meters per second')
print('The initial condition in the y-direction is v_iy = ' '%0.3f' % v_iy, 'meters per second')

The initial condition in the x-direction is v_ix = 3.277 meters per second
The initial condition in the y-direction is v_iy = 2.294 meters per second


The time it takes the object to reach the maximum height and the maximum height of the projectile

In [7]:
t_max = (-v_iy/a_y)
print('The time it takes to reach the maximum height is t_max = ' '%0.3f' % t_max, 'seconds') 
y_max = (y_i) + (v_iy*t_max)+(0.5*a_y*t_max**2)
print('The maximum height reached by the object is y_max = ' '%0.3f' % y_max, 'meters')

The time it takes to reach the maximum height is t_max = 0.234 seconds
The maximum height reached by the object is y_max = 40.268 meters


The time it takes for the object to hit the ground, the horizontal distance before hitting the ground and the final velocity when the object hits the ground

In [12]:
t_f1 = (-v_iy - np.sqrt(v_iy**2+2*a_y*(-y_i)))/(a_y)
t_f2 = (-v_iy + np.sqrt(v_iy**2+2*a_y*(-y_i)))/(a_y)
print('Potential times it takes to hit the ground')
print('time 1 = ' '%0.3f' % t_f1, 'seconds and time 2 = ' '%0.3f' % t_f2,' seconds')
print('remember we need for the time to be positive')

Potential times it takes to hit the ground
time 1 = 3.099 seconds and time 2 = -2.631  seconds
remember we need for the time to be positive


Therefore, we only accept the first time (positive) $t=3.099$ seconds.

In [18]:
t_f = t_f1 # because t_f1 is positive

Range = v_ix * t_f 
print('The horizontal range of the object before hitting the group is: ' '%0.3f' %  Range, 'meters')
v_fx = v_ix
v_fy = v_iy +a_y*t_f

v_f = np.sqrt(v_ix**2+v_fy**2)
print('The final velocity of the object when it hits the ground is: ' '%0.3f' % v_f, 'meters per second downward')

The horizontal range of the object before hitting the group is: 10.155 meters
The final velocity of the object when it hits the ground is: 28.298 meters per second downward
